src/libs/actions/Report.js

```diff
@@ -754,16 +757,6 @@ function fetchAllReports(
             // We are waiting 8 seconds since this provides a good time window to allow the UI to finish loading before
             // bogging it down with more requests and operations.
             }, shouldDelayActionsFetch ? 8000 : 0);
-
-            // Update currentlyViewedReportID to be our first reportID from our report collection if we don't have
-            // one already.
-            if (!shouldRedirectToReport || lastViewedReportID) {
-                return;
-            }
-
-            const firstReportID = _.first(reportIDs);
-            const currentReportID = firstReportID ? String(firstReportID) : '';
-            Onyx.merge(ONYXKEYS.CURRENTLY_VIEWED_REPORTID, currentReportID);
```

Comment:
This is no longer needed because:
1. It was only called like that in `AuthScreens`
2. Initial report ID would be derived in `MainDrawerNavigator`
3. `MainDrawerNavigator` would mount the `ReportScreen`
4. `ReportScreen` would set `currentReportID` during mount and updates 
    https://github.com/Expensify/Expensify.cash/blob/42b73072b87ff9e22e2ce126c9b825c4096f6c50/src/pages/home/ReportScreen.js#L32-L35

Reply:
Ok so we will stop redirecting to the report and instead just show the most recently accessed report. Seems like a nice clean solution. Redirecting from here always felt a bit weird tbh.